base/broadcast.jl

```diff
@@ -10,21 +10,9 @@ import Base: broadcast, broadcast!
 export bitbroadcast, dotview
 export broadcast_getindex, broadcast_setindex!
 
-## Broadcasting utilities ##
-
-broadcast_array_type() = Array
```

Comment:
I think we can avoid defining these if we delete the method in line 26 and just use [this method](https://github.com/JuliaLang/julia/blob/dbbd7e510d54810071d81acfd849e657bbb36dbd/base/broadcast.jl#L465) which was the origin purpose of this PR.